extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -201,6 +201,46 @@ In the following example, we display a `toolbar` if the window width is less tha
 See live demos at [AMP By Example](https://ampbyexample.com/components/amp-sidebar/).
 {% endcall %}
 
+## Sidebar for Stories
+The sidebar is also usable within the `amp-story` [component](https://www.ampproject.org/stories/). However there are certain restrictions on what attributes and features are allowed. The following are **disallowed** attributes and features of sidebar within an `amp-story`:
+
+- side
+- open
+- toolbar
+- toolbar-target
+
+### Behavior
+- The `<amp-sidebar>` must be a direct child of `<amp-story>`
+- The sidebar defaults to the "start" side for regular AMP documents, meaning right for left-right languages and left for right-to-left languages
+- The `<amp-sidebar>` has default background color of white and is overridable in CSS
+- The `<amp-sidebar>` the maximum width of the sidebar is enforced (320px for the desktop experience and otherwise 280px)
+- By default, a 'hamburger' style button that opens/closes the sidebar will appear on the story UI
+
+*Example: Basic Sidebar in a Story*
+
+In the following example, we've placed a simple sidebar in within a story. Remember that you will need to link to both the `amp-sidebar` and `amp-story` scripts in the head of the document in order to use these components together.
```

Comment:
"In the following example, we've placed a simple sidebar in within a story. Remember that you will need to link to both the `amp-sidebar` and `amp-story` scripts in the head of the document in order to use these components together."

to

"The following examples shows a simple `<amp-sidebar>` within an `<amp-story>` component."

I think we can cut out that the script tags needed to be added to the head of the document.

Reply:
Done.